src/m365/spo/commands/contenttype/contenttype-add.ts

```diff
@@ -68,16 +70,30 @@ class SpoContentTypeAddCommand extends SpoCommand {
 
         return request.post(requestOptions);
       })
-      .then((res: string): void => {
+      .then((res: string): Promise<void> => {
         const json: ClientSvcResponse = JSON.parse(res);
         const response: ClientSvcResponseContents = json[0];
         if (response.ErrorInfo) {
           cb(new CommandError(response.ErrorInfo.ErrorMessage));
```

Comment:
Let's see if we can avoid this to have a single exit point in the command. Multiple exit points make maintaining the code harder.

Reply:
ok I think its ok now 😋. Anyway it is as it was before 👍